multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/heancms/HeanCms.kt

```diff
@@ -182,9 +181,9 @@ abstract class HeanCms(
         return MangasPage(mangaList, hasNextPage = false)
     }
 
-    private fun getSearchMangas(vSlug: String, vTitle: String): SManga {
-        val vThumbnail = seriesSlugMap?.get(vSlug)?.get(1) ?: ""
-        val vStatus = seriesSlugMap?.get(vSlug)?.get(2) ?: ""
+    private fun createSearchResult(vSlug: String, vTitle: String): SManga {
```

Comment:
Can remove the `v` from the parameter names.

```suggestion
    private fun createSearchResult(slug: String, title: String): SManga {
```

Reply:
I forgot and changed it in the other commit